server/repository/src/migrations/v1_01_11/mod.rs

```diff
@@ -1,6 +1,7 @@
 use super::{version::Version, Migration};
 mod activity_log;
 mod is_sync_updated_for_requisition;
+mod name_indexes;
```

Comment:
Develop is currently at v11 already.. so these should be in the v1.1.12 migration 😕

Reply:
We don't know a definite patient program version yet, for this reason could we please use the diesel migrations for now? The problem is that if v1.1.12 gets released and deployed before the merge, this change will never run on already deployed systems... The diesel migrations will work as expected though because it goes through all migration files in chronological order and applies all migrations that haven't run yet.

Reply:
hm. possibly the v12 migration is an error then. main is on 1.1.10 - I'll make a correction for develop